test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -75,14 +75,30 @@ public async Task SingleOperationTriggerTest()
         [Fact]
         public async Task BatchSizeOverrideTriggerTest()
         {
-            const int batchSize = 20;
+            // Use enough items for the default batch size to require 4 batches but then
```

Comment:
"// Use enough items to require 4 batches to be processed" is simpler to understand.